package.json

```diff
@@ -122,6 +119,7 @@
     "rollup-plugin-sourcemaps": "^0.6.3",
     "semver": "^7.3.7",
     "terser": "5.31.0",
+    "tsx": "^4.10.3",
```

Comment:
Any reasoning on choosing `tsx` over `ts-node`? Just curious.

Reply:
`tsx` is much better maintained as `ts-node`. I am switching in all my projects to tsx now.